fenix/app/src/main/res/values/strings.xml

```diff
@@ -2068,6 +2068,152 @@
+    <!-- Clickable text from the contextual onboarding card that links to review quality check support article. -->
+    <string name="review_quality_check_contextual_onboarding_learn_more_link">Learn more</string>
+    <!-- Caption text to be displayed in review quality check contextual onboarding card above the opt-in button. First parameter is the Fakespot product name. Following parameters are for clickable texts defined in review_quality_check_contextual_onboarding_privacy_policy and review_quality_check_contextual_onboarding_terms_use. In the phrase "Fakespot by Mozilla", "by" can be localized. Does not need to stay by. -->
+    <string name="review_quality_check_contextual_onboarding_caption">By selecting “Yes, try it” you agree to %1$s by Mozilla’s %2$s and %3$s.</string>
+    <!-- Clickable text from the review quality check contextual onboarding card that links to Fakespot privacy policy. -->
+    <string name="review_quality_check_contextual_onboarding_privacy_policy">privacy policy</string>
+    <!-- Clickable text from the review quality check contextual onboarding card that links to Fakespot terms of use. -->
+    <string name="review_quality_check_contextual_onboarding_terms_use">terms of use</string>
+    <!-- Text for opt-in button from the review quality check contextual onboarding card. -->
+    <string name="review_quality_check_contextual_onboarding_primary_button_text">Yes, try it</string>
+    <!-- Text for opt-out button from the review quality check contextual onboarding card. -->
+    <string name="review_quality_check_contextual_onboarding_secondary_button_text">Not now</string>
+    <!-- Text for the first CFR presenting the review quality check feature. -->
+    <string name="review_quality_check_first_cfr_message" tools:ignore="UnusedResources">Tap the cart to give review checker a try. Know whether you can trust this product’s reviews — before you buy.</string>
+    <!-- Text for the second CFR presenting the review quality check feature. -->
+    <string name="review_quality_check_second_cfr_message" tools:ignore="UnusedResources">Are these reviews reliable? Open review checker to find out fast. Get an adjusted rating with unreliable reviews removed.</string>
+    <!-- Browser menu button to open review quality check -->
+    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <!-- Browser menu button content description to close review quality check-->
+    <string name="browser_menu_review_quality_check_close">Close review quality check</string>
+    <!-- Content description for review quality check star rating. -->
+    <string name="review_quality_check_star_rating_content_description">%1$s out of 5 stars</string>
+    <!-- Text for the review quality check highlights card minimize button. -->
```

Comment:
Seems like the remaining comments in strings could add more context, in the same way that all the other comments were updated